src/game_api/rpc.cpp

```diff
@@ -565,71 +504,65 @@ std::vector<uint32_t> filter_entities(std::vector<uint32_t> entities, std::funct
 
 std::vector<uint32_t> get_entities()
 {
-    return get_entities_by({}, 0, LAYER::BOTH);
+    return get_entities_by(0, 0, LAYER::BOTH);
 }
 
 std::vector<uint32_t> get_entities_by_layer(LAYER layer)
 {
-    return get_entities_by({}, 0, layer);
+    return get_entities_by(0, 0, layer);

```

Comment:
This might be slightly more readable, but it allocates memory at no additional functionality. So please revert this, unless I missed some subtle details.